src/sentry/static/sentry/app/views/settings/incidentRules/utils/hasThresholdValue.tsx

```diff
@@ -0,0 +1,12 @@
+import {defined} from 'app/utils';
+
+/**
+ * A threshold has a value if it is not one of the following:
+ *
+ * '', null, undefined
+ *
+ *
+ */
+export default function hasThresholdValue(value: number | '' | null): value is number {
```

Comment:
maybe: 
```suggestion
export default function hasThresholdValue<V extends number | '' | null>(value: V): number { 
```

Reply:
This is not equivalent, the former is a [type guard](https://www.typescriptlang.org/docs/handbook/advanced-types.html#user-defined-type-guards) to narrow the types during run time